ML/PYTORCH/README.md

```diff
@@ -0,0 +1,33 @@
+This tutorial presents an example of using a pre-trained Pytorch model in the AMReX framework.
+A model is included as an example (`/Exec/model.pt`).
+
+You will first need to download the appropriate Pytorch library (libtorch) in the current directory. Note that we have renamed the folder to indicate whether it uses CPU or CUDA.
+
+```shell
+# CPU only
+wget https://download.pytorch.org/libtorch/cpu/libtorch-cxx11-abi-shared-with-deps-1.9.0%2Bcpu.zip
+unzip libtorch-cxx11-abi-shared-with-deps-1.9.0+cpu.zip
+mv libtorch libtorch_cpu
+
+# CUDA 11.1
+wget https://download.pytorch.org/libtorch/cu111/libtorch-cxx11-abi-shared-with-deps-1.9.0%2Bcu111.zip
+unzip libtorch-cxx11-abi-shared-with-deps-1.9.0%2Bcu111.zip
```

Comment:
```suggestion
unzip libtorch-cxx11-abi-shared-with-deps-1.9.0+cu111.zip
```

Reply:
^ this was the actual file that it downloaded.  Does it do the same for you?

Reply:
Ah that was a typo